napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -303,7 +303,9 @@ def __init__(self, parent, title: str = '', vertical=False):
             # dock widget.  If it is, then add the close button option to the title bar.
             actions = [
                 action.text()
```

Comment:
Could we maybe make this two lines like

```python
plugins_menu = self.parent().parent()._qt_viewer.viewer.window.plugins_menu
actions = [action.text() for action in plugins_menu.actions()]
```
or
```python
window = self.parent().parent()._qt_viewer.viewer.window
actions = [action.text() for action in window.plugins_menu.actions()]
```
I find it a little hard to read as it is now given the black formatting

Reply:
sure thing.  done!

Reply:
actually, I see it still reformatted the `plugins_menu` line.  I am changing it again.